test/parallel/test-process-binding-internalbinding-allowlist.js

```diff
@@ -6,17 +6,12 @@ const assert = require('assert');
 
 // Assert that allowed internalBinding modules are accessible via
 // process.binding().
-assert(process.binding('async_wrap'));
```

Comment:
Souldn't we keep this test, but invert it to confirm that it is indeed no longer exposed?